scripts/main.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+
+from analyst_collective import Credentials, Runner
+import argparse, json, sys, os
+
+parser = argparse.ArgumentParser(description='Analyst Collective Runner')
+parser.add_argument('--credentials', default="dbcredentials.txt", type=str, help='Path to database credentials file')
```

Comment:
seems weird to have these two config files default to living in two different directories